src/System.Text.Json/src/Configurations.props

```diff
@@ -2,12 +2,13 @@
   <PropertyGroup>
     <PackageConfigurations>
       netstandard;
-      netcoreapp;
+      netcoreapp3.0;
```

Comment:
I believe this can target netstandard2.1. I don't think we rely on anything that is netcoreapp3.0 specific (which wouldn't already be in netstandard2.1). What would be the benefit of doing this, btw?

Reply:
Is it worth it?

Reply:
Yep, just realized Rune isn't in netstandard2.1: https://github.com/dotnet/standard/issues/1417